package.json

```diff
@@ -66,18 +66,10 @@
     "node": "10"
   },
   "dependencies": {
-    "@signalk/freeboard-sk": "^1.0.0",
-    "@signalk/instrumentpanel": "0.x",
-    "@signalk/maptracker": "^1.0.0",
-    "@signalk/playground": "^1.0.0",
-    "@signalk/sailgauge": "^1.1.0",
+    "@serialport/bindings": "^8.0.6",
     "@signalk/server-admin-ui": "~1.15.0",
-    "@signalk/set-system-time": "^1.2.0",
     "@signalk/signalk-schema": "1.3.1",
-    "@signalk/signalk-to-nmea0183": "^1.0.0",
-    "@signalk/simplegauges": "^1.0.1",
-    "@signalk/streams": "~1.4.0",
-    "@signalk/zones": "^1.0.0",
+    "@signalk/streams": "^1.4.0",
```

Comment:
Is change from ^ to ~ intentional? I think ~ "freeze major and minor" is more appropriate for us here.